webrecorder/browsertrix
# Update archived item URLs

Partially addresses https://github.com/webrecorder/browsertrix-cloud/issues/935



### Changes

Changes to URLs in "Crawling", "All Archived Items", and "Collections":
```diff
  /workflows/crawls
  /workflows/crawls/<workflow id>
- /workflows/crawls/<workflow id>#artifacts
+ /workflows/crawls/<workflow id>#crawls
  /workflows/crawls/<workflow id>#watch
  /workflows/crawls/<workflow id>#settings
  /workflows/crawls/<workflow id>?edit
- /workflows/crawl/<workflow id>/artifact/<crawl id>
- /artifacts/crawls
+ /items
- /artifacts/crawls?artifactType=crawl
+ /items/crawl
- /artifacts/crawls?artifactType=upload
+ /items/upload
  /collections
  /collections/edit/<coll id>
  /collections/view/<coll id>
  /collections/view/<coll id>/items
- /collections/view/<coll id>/artifact/<item id>
```

This standardizes the archived item detail view by removing the separate `/workflows/crawl/<workflow id>/artifact/<crawl id>`  view.

Also adds a redirect from `/artifacts/upload/<upload id>` to handle links from ArchiveWeb.Page.

### Manual testing

1. Log in and test navigation in Crawling, All Archived Items, and Collections. Verify navigation works as expected down to archived item details.
2. Copy an upload ID and go to `http://localhost:9870/orgs/<org id>/artifacts/upload/<upload id>`. Verify redirect to archived item detail
